frontend/src/components/mine/Tailings/MineTailingsInfo.js

```diff
@@ -212,10 +201,7 @@ export class MineTailingsInfo extends Component {
                     </Col>
                     <Col id={`status-${id}`} span={4}>
                       <h6 className={isOverdue ? "bold" : null}>
-                        <DocumentStatusText
-                          doc={doc}
-                          expectedDocumentStatusOptions={this.props.expectedDocumentStatusOptions}
-                        />
+                        {doc ? doc.exp_document_status.description : String.LOADING}
```

Comment:
Could we add this constant to `frontend-public` as well?

Reply:
Fixed, thank you